src/main/java/core/selenium/WebElementAction.java

```diff
@@ -62,6 +60,17 @@ public WebElement getWebElementByXpathAndValue(final String xpath, final String
         return driver.findElement(By.xpath(String.format(xpath, value)));
     }
 
+    /**
+     * Gets the web element with xpath and a value to replace in it.
+     *
+     * @param xpath the element's xpath
+     * @param value the value to replace the xpath
+     * @return a web element with the provided features
+     */
+    public WebElement getWebElementByCssAndValue(final String xpath, final String value) {
```

Comment:
the method name says CSS but the variable's name is xpath. Maybe you should change that.

Reply:
Okay, already changed.